mp/src/game/server/particle_system.cpp

```diff
@@ -180,9 +187,14 @@ void CParticleSystem::Activate( void )
 //-----------------------------------------------------------------------------
 // Purpose: 
 //-----------------------------------------------------------------------------
-void CParticleSystem::StartParticleSystemThink( void )
+void CParticleSystem::Think( void )
 {
-	StartParticleSystem();
```

Comment:
I think this was used to delay starting a particle system. Was there a particular reason to remove it?

Reply:
It was never called previously, and there is no such delay in `info_particle_system` AFAIK.